cli/core/findStalePods.go

```diff
@@ -25,8 +27,20 @@ type Cache struct {
 	PodOwnerShares map[string]PodOwnerShare
```

Comment:
This cache only lasts for the duration of a CLI command, right?

Reply:
yep